README.md

```diff
@@ -1,2 +1,42 @@
 Memoization decorator for python 
-Store function results on disk. Load if call to the function was previously stored. 
+=== 
+Add @Memoized before function definition for automatic caching. 
```

Comment:
1. `@Memoized`, not  @Memoized 
2. The first thing should be a dead-simple example IMO, not explanations.

Reply:
1. I think you fixed the capitalization already. 
2. Looks good.